### ISSUE TITLE
process_map and render=FALSE ignores layout specification

### ISSUE THREAD

**Issue by kkmann** [bupaverse/processmapR#27]
Hey,

I bumped into this while trying to export my maps to pdf via 'export_graph'. It works fine but ignores custom layout specifications.

**Comment by fmannhardt** [comment]
I assume you are taking about the parameters generated with `layout_pm`. Indeed, this is a limitation as we are unable to set the relevant DOT attributes via the DiagrammeR library. I already created an issue upstream here:
https://github.com/rich-iannone/DiagrammeR/issues/360

For now, there is little we can do about this (except patching DiagrammeR, which I would like to avoid). We should definitely document this, but I had hoped that DiagrammeR would be updated. 

You may have some success by manually applying our workaround:
```
	  	# hack to add 'weight' attribute to the graph
		graph$edges_df %>%
			mutate(len = weight, decorate = constraint) -> graph$edges_df

		graph %>% render_graph() -> graph

		graph$x$diagram %>%
			stringr::str_replace_all("len", "weight") %>%
			stringr::str_replace_all("decorate", "constraint") -> graph$x$diagram
```

**Comment by kkmann** [comment]
Yes, exactly, makes it a bit tricky to produce high quality pdf graphs for publications. Thanks for raising that with DiagrammR!
